pkg/command/service/measure.go

```diff
@@ -494,8 +496,20 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
 				"ingress_config_ready",
 				"ingress_lb_ready"}}, rawRows...)
 			total := measureFinalResult.readyCount + measureFinalResult.notReadyCount + measureFinalResult.notFoundCount + measureFinalResult.failCount
+			knativeVersion := getKnativeVersion(p)
+			ingressInfo := getIngressController(p)
+			endtime := time.Now()
 			if measureFinalResult.readyCount > 0 {
 				fmt.Printf("-------- Measurement --------\n")
+				fmt.Printf("Basic Information:\n")
+				fmt.Printf("  Start Time: %v\n", starttime.Format("2006-01-02 15:04:05"))
+				fmt.Printf("  End Time: %v\n", endtime.Format("2006-01-02 15:04:05"))
```

Comment:
@gyohuangxin 
One question is,  what the `Start Time` and `End Time` are for?
If the user want to know how much time `kperf` CLI executed, it can be done by `time kperf xxxx`

Reply:
The start time and end time are the measurement start and end time. So it's duplicated? If so, I will remove it, Thanks.